Task 2348 create debug app

srikanth-vg opened the pull request:
Implemented:

- To enable debug mode, the user needs to tap 7 times on the version number 
- Using "Exit debug mode", the user can disable the debug mode.
- In normal mode, the app will point to production 
- Now we are showing only the app version number in the login screen

Task: https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/2348




https://github.com/user-attachments/assets/68eb7dc6-9e01-43bb-a6f5-663350418d94

srikanth-vg committed:
Created debug app

- Updated debug mode bundle ID to tech.opentoall.aviv.scoutroute.debug
- Added debug mode App icon
- Added debug banner on the app screens and it will be visible only in debug mode
- Updated debug mode Google Services plist file with new bundle ID

srikanth-vg committed:
Updated the debug version to 1.0(3.1)

srikanth-vg committed:
updated debug label and added to the bottom of the screen

uw-amy-bordenave commented:
Rejected initial implementation on premise - rather than separate apps, the decision has been made (in 2025-09-15 dev meeting) to instead implement a debug menu into the app.
Refer to the latest comments on [2348](https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/2348) for more information.

srikanth-vg committed:
reverted debug mode app changes

srikanth-vg committed:
Revert "Updated the debug version to 1.0(3.1)"

This reverts commit 235440c4621072659123b8c3fa2f411b920cad95.

srikanth-vg committed:
Implemented enabling and disabling debug mode.

- 7 taps on version number will prompt to enable the debug mode
- by clicking on exit from debug button to prompt disable mode

srikanth-vg committed:
Updated version no to 1.0(3) to 1.0.1(1)

srikanth-vg committed:
removed displaying build number in login screen
